web/source/osk/languageMenu.ts

```diff
@@ -433,12 +439,22 @@ namespace com.keyman.osk {
       }
 
       const unlockBodyScroll = () => {
+        if(!this.originalBodyStyle) {
+          // We shouldn't be able to reach here, but in case things go out-of-order due
+          // to some unforeseen circumstance, let's null-guard here.
+          return;
```

Comment:
Should we console.error this so we can track it on Sentry, given we don't currently expect this to happen?